2023-03-25 22:46:40,815 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Configure stats pid to 28244
2023-03-25 22:46:40,816 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Loading settings from C:\Users\inigo\.config\wandb\settings
2023-03-25 22:46:40,816 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Loading settings from F:\CV_MATERIAL\M6\WEEK3\detr\wandb\settings
2023-03-25 22:46:40,817 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-03-25 22:46:40,817 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-03-25 22:46:40,817 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': '..\\traindetr.py', 'program': 'f:\\cv_material\\m6\\week3\\traindetr.py'}
2023-03-25 22:46:40,819 INFO    MainThread:28244 [wandb_init.py:_log_setup():506] Logging user logs to F:\CV_MATERIAL\M6\WEEK3\detr\wandb\run-20230325_224640-4mn0p3nk\logs\debug.log
2023-03-25 22:46:40,820 INFO    MainThread:28244 [wandb_init.py:_log_setup():507] Logging internal logs to F:\CV_MATERIAL\M6\WEEK3\detr\wandb\run-20230325_224640-4mn0p3nk\logs\debug-internal.log
2023-03-25 22:46:40,820 INFO    MainThread:28244 [wandb_init.py:_jupyter_setup():452] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000228FFA7A4F0>
2023-03-25 22:46:40,822 INFO    MainThread:28244 [wandb_init.py:init():546] calling init triggers
2023-03-25 22:46:40,823 INFO    MainThread:28244 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {}
2023-03-25 22:46:40,823 INFO    MainThread:28244 [wandb_init.py:init():568] re-initializing run, found existing run on stack: 58usy3dw
2023-03-25 22:46:40,825 INFO    MainThread:28244 [wandb_run.py:_finish():1852] finishing run mcv-m5-team4/detrFineTune/58usy3dw
2023-03-25 22:46:40,826 INFO    MainThread:28244 [wandb_run.py:_atexit_cleanup():2081] got exitcode: 0
2023-03-25 22:46:40,827 INFO    MainThread:28244 [wandb_run.py:_restore():2064] restore
2023-03-25 22:46:40,827 INFO    MainThread:28244 [wandb_run.py:_restore():2070] restore done
2023-03-25 22:46:45,731 INFO    MainThread:28244 [wandb_run.py:_footer_history_summary_info():3429] rendering history
2023-03-25 22:46:45,732 INFO    MainThread:28244 [wandb_run.py:_footer_history_summary_info():3461] rendering summary
2023-03-25 22:46:45,749 INFO    MainThread:28244 [wandb_run.py:_footer_sync_info():3387] logging synced files
2023-03-25 22:46:45,757 INFO    MainThread:28244 [wandb_init.py:init():602] starting backend
2023-03-25 22:46:45,758 INFO    MainThread:28244 [wandb_init.py:init():606] setting up manager
2023-03-25 22:46:45,777 INFO    MainThread:28244 [wandb_init.py:init():613] backend started and connected
2023-03-25 22:46:45,785 INFO    MainThread:28244 [wandb_run.py:_label_probe_notebook():1202] probe notebook
2023-03-25 22:46:45,785 INFO    MainThread:28244 [wandb_run.py:_label_probe_notebook():1212] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-25 22:46:45,786 INFO    MainThread:28244 [wandb_init.py:init():701] updated telemetry
2023-03-25 22:46:45,884 INFO    MainThread:28244 [wandb_init.py:init():741] communicating run to backend with 60.0 second timeout
2023-03-25 22:46:47,240 INFO    MainThread:28244 [wandb_run.py:_on_init():2131] communicating current version
2023-03-25 22:46:47,514 INFO    MainThread:28244 [wandb_run.py:_on_init():2140] got version response upgrade_message: "wandb version 0.14.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-25 22:46:47,515 INFO    MainThread:28244 [wandb_init.py:init():789] starting run threads in backend
2023-03-25 22:46:47,808 INFO    MainThread:28244 [wandb_run.py:_console_start():2112] atexit reg
2023-03-25 22:46:47,809 INFO    MainThread:28244 [wandb_run.py:_redirect():1967] redirect: SettingsConsole.OFF
2023-03-25 22:46:47,812 INFO    MainThread:28244 [wandb_init.py:init():831] run started, returning control to user process
2023-03-25 22:48:27,757 INFO    MainThread:28244 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-03-25 22:48:27,770 INFO    MainThread:28244 [wandb_init.py:_pause_backend():417] pausing backend
2023-03-25 22:48:30,072 WARNING MsgRouterThr:28244 [router.py:message_loop():77] message_loop has been closed
